pkgs/development/compilers/gerbil/gerbil-tools.nix

```diff
@@ -0,0 +1,53 @@
+{ pkgs, gccStdenv, callPackage, fetchFromGitHub, gambit, gambit-unstable }:
+
+let stdenv = gccStdenv; in # See ../gambit/build.nix
+
+rec {
+  gerbilPackage = {
```

Comment:
Should the two `gerbilPackage` definitions be unified and just called with different arguments if needed?

Reply:
Oops, I forgot to delete `gerbil-tools` after renaming it to `gerbil-support` (because there's a `gerbil-tools` package in the gerbil-directory).